supertokens_python/recipe/thirdparty/emaildelivery/services/backward_compatibility/__init__.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Dict, Union
+
+from supertokens_python.ingredients.emaildelivery.types import \
+    EmailDeliveryInterface
+from supertokens_python.recipe.emailverification.emaildelivery.service.backward_compatibility import \
+    BackwardCompatibilityService as EVBackwardCompatibilityService
+from supertokens_python.recipe.emailverification.types import User
+from supertokens_python.recipe.thirdparty.interfaces import RecipeInterface
+from supertokens_python.recipe.thirdparty.types import \
+    TypeThirdPartyEmailDeliveryInput
+from supertokens_python.supertokens import AppInfo
+
+if TYPE_CHECKING:
+    from supertokens_python.recipe.thirdparty.utils import \
+        InputEmailVerificationConfig
+
+
+class BackwardCompatibilityService(EmailDeliveryInterface[TypeThirdPartyEmailDeliveryInput]):
+    def __init__(self,
+                 app_info: AppInfo,
+                 recipeInterfaceImpl: RecipeInterface,
+                 email_verification_feature: Union[InputEmailVerificationConfig, None] = None,
+                 ) -> None:
+        self.app_info = app_info
```

Comment:
no need to make this an instance variable

Reply:
Removed.